src/ffi/testing/testing.cc

```diff
@@ -282,6 +284,86 @@ TVM_FFI_STATIC_INIT_BLOCK() {
 namespace tvm {
 namespace ffi {
 
+// -----------------------------------------------------------------------------
+// Implementation functions for schema testing
+// These are registered via both GlobalDef and TVM_FFI_DLL_EXPORT_TYPED_FUNC
+// -----------------------------------------------------------------------------
+namespace schema_test_impl {
+
```

Comment:
move these tests files into a cpp.load_inline test instead, so we don't have to blow up the testing. Given we use the same mechanism as the global one, we don't need to test so many functions